recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
 # Based on recipes from birdhouse and anaconda channel.
 
 {% set name = "openjdk" %}
-{% set version = "8.0.144" %}
-{% set zulu_build = "8.23.0.3" %}
-{% set build_number = 2 %}
-{% set sha256_linux = "7e6284739c0e5b7142bc7a9adc61ced70dc5bb26b130b582b18e809013bcb251" %}
-{% set sha256_osx = "ff533364c9cbd3b271ab5328efe28e2dd6d7bae5b630098a5683f742ecf0709d" %}
-{% set sha256_win = "f1d9d3341ef7c8c9baff3597953e99a6a7c64f8608ee62c03fdd7574b7655c02" %}
+{% set version = "9.0.7" %}
+{% set zulu_build = "9.0.7.1" %}
+{% set build_number = 0 %}
+{% set sha256_linux = "45f2dfbee93b91b1468cf81d843fc6d9a47fef1f831c0b7ceff4f1eb6e6851c8" %}
+{% set sha256_osx = "4b1f8529ff3a8bebc974e2a22395cb27ad8750e386c8c4d1b0a1b16f89cfcf66" %}
+{% set sha256_win = "75f76c53c6a1f12b1a571b86bd9708ab75adf582d689dddc94fdd77dcc0f3f5c" %}
```

Comment:
Please move the sha256 sums into the body of the meta.yaml to confirm with the new style. Hopefully this will allow the regro-cf-autotick-bot to create PRs for future updates.